registry/bin/boot

```diff
@@ -40,9 +40,20 @@ function etcd_set_default {
   set -e
 }
 
+function etcd_safe_mkdir {
```

Comment:
There was a call in one of the units to `etcdctl mkdir /deis/cache`. We do this in the other boot scripts, so I refactored that into registry's boot script. Since there was no call to `etcd_safe_mkdir` in there I just copy-pasta'd the function over.